_tools/check_deprecation.ts

```diff
@@ -0,0 +1,105 @@
+// Copyright 2022-2022 the Deno authors. All rights reserved. MIT license.
+
+import { VERSION } from "../version.ts";
+import * as semver from "../semver/mod.ts";
+
+const EXTENSIONS = [".mjs", ".js", ".ts", ".rs"];
+const EXCLUDED_PATHS = [
+  ".git",
+  "node/_module",
+  "dotenv/testdata",
+  "fs/testdata",
+  "http/testdata",
+  "node/_module/cjs",
+  "node/_module/node_modules",
+  "node/_tools",
+  "node/testdata",
+];
```

Comment:
I think we should add `fs/exists.ts` to `EXCLUDED_PATHS` as there's some controversies about its deprecation.

Reply:
Ok, I'll put in on the list and log a warning, so it is not forgotten about.